balances-adjustment/src/main/java/com/powsybl/balances_adjustment/util/AbstractInjectionBasedArea.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2025, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
```

Comment:
I think SPDX identifiers are missing in the header
```suggestion
 * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  * SPDX-License-Identifier: MPL-2.0
```